packages/server/src/db.js

```diff
@@ -5,4 +5,21 @@ const knexfile = require('../db/knexfile');
 
 const knex = Knex(knexfile.development);
 
-module.exports = knex;
+class LocationRepository {
+  static async getLocations() {
+    return knex.select().from('locations');
+  }
+
+  static async addLocation(payload) {
+    return knex('locations').insert({
+      lat: payload.lat,
+      lon: payload.lon,
+    });
+  }
+
+  static async latestLocation() {
+    return knex.select().from('locations').orderBy('created_at', 'desc').limit(1);
+  }
+}
+
+module.exports = LocationRepository;
```

Comment:
You might want to change this to be `module.exports = { LocationRepository };` That way it'll be less refactoring if you want to export something else in this file.

Or, you could also just break the db stuff into separate files.

I'll leave it up to you if you want to make this change, just because there might not be anything else to export.

---

Also, (and this suggestion might actually be worse) if you want you can use a normal object, since you're just using static methods.

```js
const LocationRepository = {
  async getLocations() {
    // ...
  },
  // ...
};
```

Realistically they're pretty much the same, except a class has a bit more overhead. However, that overhead is pretty inconsequential, especially in an environment outside of the browser.